hawkbit-repository/hawkbit-repository-jpa/src/test/java/org/eclipse/hawkbit/repository/jpa/TargetManagementTest.java

```diff
@@ -667,11 +763,14 @@ public void bulkTargetCreationAndDelete() throws Exception {
 
         targetManagement.delete(targetsIdsToDelete);
 
-        final List<Target> targetsLeft = targetManagement.findAll(PageRequest.of(0, 200)).getContent();
+        final List<Target> targetsLeft = targetQueryExecutionManagement.findAll(PageRequest.of(0, 200))
+                .getContent();
         assertThat(firstList.spliterator().getExactSizeIfKnown() - numberToDelete).as("Size of split list")
                 .isEqualTo(targetsLeft.spliterator().getExactSizeIfKnown());
 
-        assertThat(targetsLeft).as("Not all undeleted found").doesNotContain(deletedTargets);
+        assertThat(targetsLeft).as("Not all undeleted found").doesNotContainAnyElementsOf(targetsToDelete);
+
+        targetsLeft.containsAll(targetsIdsToDelete);
```

Comment:
Missing assertion. Is this what you wanna check?
```suggestion
        assertThat(targetsLeft).extracting(Target::getId).isNotIn(targetsIdsToDelete);
```